app/assets/stylesheets/modules/sul-icons.scss

```diff
@@ -18,31 +23,31 @@ $sul-icon-prefix: 'sul-icon';
   @extend .fa-rss-square;
 
   color: $sul-rss-icon-color;
-  margin-right: .3em;
+  margin-right: 0.3em;
 }
 
 // Style icons in facets
-.facet-values .#{$sul-icon-prefix}, .facet-values .#{$sul-icon-prefix}.blacklight-icons {
-  @extend .#{$sul-icon-prefix}-1-5x;
-  align-items: center;
-  color: $black;
-  display: table-cell;
-  width: 30px;
-  &::before, svg {
-    height: 21px;
-    width: 21px;
-    vertical-align: top;
+.facet-values {
+  .blacklight-icons {
```

Comment:
You need to use the .sul-icon selector of there is a problem with the `x` on selected facets.

<img width="343" alt="Screenshot 2025-04-28 at 4 15 31 PM" src="https://github.com/user-attachments/assets/77b74e2d-5437-4952-b7c3-05826ea68db4" />

Reply:
Thank you. I'll look at that.

Reply:
This is fixed  up.